internal/compile/compile.go

```diff
@@ -113,7 +114,7 @@ const (
 	CONSTANT    //                - CONSTANT<constant>  value
 	MAKETUPLE   //        x1 ... xn MAKETUPLE<n>        tuple
 	MAKELIST    //        x1 ... xn MAKELIST<n>         list
-	MAKEFUNC    //      args kwargs MAKEFUNC<func>      fn
+	MAKEFUNC    //  deflts freevars MAKEFUNC<func>      fn
```

Comment:
Change to `defaults`. `deflts` is a little too terse.

Reply:
Done.